bitnami/nats/templates/configmap.yaml

```diff
@@ -31,9 +31,9 @@ data:
     {{- end }}
 
     # Logging options
-    debug: {{ .Values.debug.enabled }}
-    trace: {{ .Values.debug.trace }}
-    logtime: {{ .Values.debug.logtime }}
+    debug: {{ ternary "true" "false" (or .Values.debug.enabled .Values.diagnosticMode.enabled) }}
+    trace: {{ ternary "true" "false" (or .Values.debug.trace .Values.diagnosticMode.enabled) }}
+    logtime: {{ ternary "true" "false" (or .Values.debug.logtime .Values.diagnosticMode.enabled) }}
```

Comment:
Do we want to enable all this at the same time ?

Reply:
Not a strong opinion, just adding as much verbosity as possible in this diagnostic mode, but to be honest I don't know what this means for NATS and if it is going to be more information than the desired one